crowbar_framework/app/models/service_object.rb

```diff
@@ -72,12 +72,12 @@ def validation_error message
   end
 
   def simple_proposal_ui?
-    proposals = Proposal.where(barclamp: "crowbar")
+    proposal = Proposal.find_by(barclamp: "crowbar")
 
     result = false
-    unless proposals[0].nil? or proposals[0]["attributes"].nil? or proposals[0]["attributes"]["crowbar"].nil?
-      if not proposals[0]["attributes"]["crowbar"]["simple_proposal_ui"].nil?
-        result = proposals[0]["attributes"]["crowbar"]["simple_proposal_ui"]
+    unless proposal.nil? or proposal["attributes"].nil? or proposal["attributes"]["crowbar"].nil?
+      if not proposal["attributes"]["crowbar"]["simple_proposal_ui"].nil?
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)<br>Style/Not: Use `!` instead of `not`. (https://github.com/bbatsov/ruby-style-guide#bang-not-not)